### ISSUE TITLE
Behaviour of draggableRange and onlyBindHandles.

### ISSUE THREAD

**Issue by alfredott** [angular-slider/ng5-slider#80]
Hello !

Thanks a lot for your plugin.

For now, I'm using your range slider with the following options (and some others that are not the interesting in this issue) :
```
{
pushRange : true,
draggableRange: true
}
```
I'm experiencing a little "glitch" when I try to drag the range :
- I start by clicking between both handles, which moves the nearest handle just under my cursor
- Before releasing the mouse click, I'm dragging my mouse left and right, which puts that handle back to its first place, and now makes the range move

The handle makes a little "flash"
[Stackblitz](https://stackblitz.com/edit/ng5-slider-range-slider-example-2p7wsa?file=src/app/app.component.ts) to reproduce this "glitch".

I perfectly understand the origin of this problem : when you first press the click button (before releasing it), the slider can't tell whether you want to move of the handles, or to drag the range.

Thus (as I saw in https://github.com/angular-slider/ng5-slider/issues/21), I then tried to use the onlyBindHandles option, so that :
- my clicks on a handle move this handle
- my clicks between the handles drag the range

However, it doesn't work that way : the onlyBindHandles option disables all clicks that are not on a handle, including the ones that I would like to see drag the range. Moreover, my mouse cursor is still "move" between the handles, which makes me think that I can drag the range.

[Stackblitz](https://stackblitz.com/edit/ng5-slider-range-slider-example-p19yhx?file=src/app/app.component.ts) to reproduce this second problem.

**TLDR :** I think that that second problem is a conflict between the "draggableRange" and "onlyBindHandles" options when they are both set to true. In that case, I would like to still be able to drag the range when clicking between the handles.
It could be solved by changing the current common behaviour of these two options.
If you prefer keeping some backwards compatibility with the old behaviour, I think it could also be solved by adding a new option. (I prefer the first solution !)

I hope you'll understand my problem and you'll find some time to look about it.

Thanks a lot, bye !

**Comment by piotrdz** [comment]
I see what you are getting at. I also don't like this behaviour, and I'll try to fix it.

I think that if you enable `draggableRange`, then the behaviour should be like you described - clicking between the handles should initiate a drag, without changing any of the handles.

As regards `onlyBindHandles`, it basically causes the slider to only bind user events to the handle elements, ignoring the slider bar. In my view, this is working as expected. Perhaps the only confusing thing is showing the movable mouse cursor, which I will fix.

It shouldn't be a difficult fix, so I should have it ready for next release.

**Comment by alfredott** [comment]
Thanks a lot for looking into my issue!

I thought the current behaviour of `draggableRange `was intended, so I tried imagining other solutions, with `onlyBindHandles `for example.

But if you think my use case should be covered by draggableRange alone, I'm perfectly OK with it :)

Good luck fixing it that, and thanks again!
Bye

**Comment by piotrdz** [comment]
Fixed now in master and will be available in next release.

**Comment by piotrdz** [comment]
Fix released with v1.2.0.

**Comment by alfredott** [comment]
Thanks !
Works properly in v1.2.1 (haven't tried v1.2.0).

Bye
